### PR TITLE
Add com.irccloud.desktop

### PR DESCRIPTION
**Pull request by barthalion**
Notified upstream [here](https://github.com/irccloud/irccloud-desktop/issues/152) but no response yet.

### CODE REVIEW

**Comment by barthalion** [comment]
bot, build com.irccloud.desktop

**Comment by barthalion** [comment]
bot, build com.irccloud.desktop

**Comment by barthalion** [comment]
bot, build com.irccloud.desktop

**Comment by barthalion** [comment]
bot, build com.irccloud.desktop

**Comment by TingPing** [comment]
I spoke with upstream months ago about me doing the same, he wasn't interested in maintaining it but didn't mind it being packaged.

File: com.irccloud.desktop.json [hunk 1]
```diff
@@ -0,0 +1,115 @@
+{
+  "app-id": "com.irccloud.desktop",
```
Comment (TingPing) [inline com.irccloud.desktop.json]: `.desktop` was historically problematic, maybe just avoid it?

File: irccloud.desktop [hunk 2]
```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Name=IRCCloud
+Comment=Chat on IRC from anywhere, and never miss a message
+Exec=irccloud
+Terminal=false
+Type=Application
+Icon=com.irccloud.desktop
+Categories=Network;IRCClient;
```
Comment (TingPing) [inline irccloud.desktop]: Probably missing `StartupWMClass`

File: com.irccloud.desktop.json (hunk 1 above)
Comment (nedrichards) [reply com.irccloud.desktop.json]: I think it's well known enough now that the issues have been shaken out (e.g. Telegram is popular) but I don't mind changing - it really depends what/if is used upstream.

Comment (barthalion) [reply com.irccloud.desktop.json]: Yeah, I assumed it generally works these days. I can change that if upstream chimes in.

**Comment by wjt** [comment]
Oh no, another app ID ending in `.desktop`…

**Comment by jwheare** [comment]
Just chiming in that say that we would be interested in co-maintaining this, see comments here: https://github.com/irccloud/irccloud-desktop/issues/152#issuecomment-584738557

**Pull request closed without merging**